playbooks/adjust-cinder-backend.yml

```diff
@@ -111,6 +117,11 @@
 
   - name: trigger stop all cinder services
     meta: flush_handlers
+    
+
+  - name: wait for cinder services down
+    shell: sleep 10s
+    when: set_cinder_host or update_cinder_backend
```

Comment:
This feels fragile. Perhaps it would be better to use a `until` loop and check the ceph API status (maybe with a sensu plugin execution). That'd be safer and more accurate.